pkg/reader/ccip.go

```diff
@@ -173,40 +173,33 @@ type RMNCurseResponse struct {
 
 // ---------------------------------------------------
 
-func (r *ccipChainReader) CommitReportsGTETimestamp(
-	ctx context.Context, ts time.Time, limit int,
-) ([]cciptypes.CommitPluginReportWithMeta, error) {
-	lggr := logutil.WithContextValues(ctx, r.lggr)
+func (r *ccipChainReader) CommitReportsGTETimestamp(ctx context.Context, ts time.Time, limit int) (cciptypes.CommitReportsByConfidenceLevel, error) {
```

Comment:
I don't think you should change the return type like this. The pattern used for `ExecutedMessages` where the confidence level is passed in as a parameter is simpler.